### PR TITLE
chore(docs): regroup benchmark pages [WIP]

### PR DESCRIPTION
**Pull request by yuxizama**
Following [Rand's suggestion](https://zama-ai.slack.com/archives/C025KGZJBGC/p1721816224640449?thread_ts=1721815136.006619&cid=C025KGZJBGC), I grouped all existing benchmarks under the "Benchmark" page in "Get Started"

I think there are some updates to do, or new benchmarks to be added (compression?), it can be done on this basis.

### CODE REVIEW

File: tfhe/docs/guides/run_on_gpu.md [hunk 1]
```diff
@@ -218,70 +218,3 @@ Depending on the platform, this can restrict the number of GPUs used to perform
 There is **nothing to change in the code to execute on multiple GPUs**, when 
 they are available and have peer access to GPU 0 via NVLink. To keep the API as user-friendly as possible, the configuration is automatically set, i.e., the user has no fine-grained control over the number of GPUs to be used.
```
Comment (agnesLeroy) [inline tfhe/docs/guides/run_on_gpu.md:220]: Here I think we can add a sentence to link to the benchmarks page. :slightly_smiling_face:

Comment (yuxizama) [reply tfhe/docs/guides/run_on_gpu.md:220]: Great idea, I added the reference for zk proof one also.

File: tfhe/docs/references/fine-grained-apis/shortint/parameters.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ fn main() {
 
 ## Impact of parameters on the operations
 
-As shown [here](../../../getting\_started/benchmarks.md), the choice of the parameter set impacts the operations available and their efficiency.
+As shown [here](../../../getting_started/benchmarks/cpu_benchmarks.md), the choice of the parameter set impacts the operations available and their efficiency.
```
Comment (IceTDrinker) [inline tfhe/docs/references/fine-grained-apis/shortint/parameters.md:37]: this probably is not relevant anymore given we use a single parameter set for CPU benchmarks

File: tfhe/docs/getting_started/benchmarks/zk_proof_benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+| 1xFheUint64  | 2.79s   | 197ms     |
+| 10xFheUint64 | 3.68s   | 251ms     |
+ 
+
+Timings in the case where the workload is mainly on the verifier, i.e., with the  `ZkComputeLoad::Verify` option.
+
+| Inputs       | Proving | Verifying |
+|--------------|---------|-----------|
+| 1xFheUint64  | 730ms   | 522ms     |
+| 10xFheUint64 | 1.08s   | 682ms     |
```
Comment (IceTDrinker) [inline tfhe/docs/getting_started/benchmarks/zk_proof_benchmarks.md:20]: for TFHE-rs: TODO we likely need to use the wasm benchmarks for the client, I think those are the bare metal ones

also TODO: size of objects

File: tfhe/docs/getting_started/benchmarks/summary.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Benchmarks
+
+This document summarizes the timings of some homomorphic operations over 64-bit encrypted integers, depending on the hardware. More details are given for [the CPU](cpu\_benchmarks.md), [the GPU](gpu\_benchmarks.md), or [zeros-knowledge proofs](zk\_proof\_benchmarks.md).
+
+### Operation time (ms) over FheUint 64
+
+{% embed url="https://docs.google.com/spreadsheets/d/1PZlnBnqax3hfo7xPir3W-et3es_Qx3Y5_CkdoQVjtb0/edit?pli=1&gid=1430012664#gid=1430012664" %}
```
Comment (IceTDrinker) [inline tfhe/docs/getting_started/benchmarks/summary.md:7]: we are embedding a google doc ?

File: tfhe/docs/getting_started/benchmarks/cpu_benchmarks.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+{% embed url="https://docs.google.com/spreadsheets/d/1yQ9etV7gNLplDkvOWaoLPeG1cO1vpbLc6zeTn0Xs8q0/edit?usp=sharing" %}
+
+All timings are based on parallelized Radix-based integer operations where each block is encrypted using the default parameters `PARAM_MESSAGE_2_CARRY_2_KS_PBS`. To ensure predictable timings, we perform operations in the `default` mode, which ensures that the input and output encoding are similar (i.e., the carries are always emptied).
+
+You can minimize operational costs by selecting from 'unchecked', 'checked', or 'smart' modes from [the fine-grained APIs](../../references/fine-grained-apis/quick\_start.md), each balancing performance and correctness differently. For more details about parameters, see [here](../../references/fine-grained-apis/shortint/parameters.md). You can find the benchmark results on GPU for all these operations [here](../../guides/run\_on\_gpu.md#benchmarks).
+
+## Programmable bootstrapping
+
+The next table shows the execution time of a keyswitch followed by a programmable bootstrapping depending on the precision of the input message. The associated parameter set is given. The configuration is Concrete FFT + AVX-512.
+
```
Comment (IceTDrinker) [inline tfhe/docs/getting_started/benchmarks/cpu_benchmarks.md:30]: missing table

File: tfhe/docs/getting_started/benchmarks/cpu_benchmarks.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+## Shortint operations
+
+The next table shows the execution time of some operations using various parameter sets of tfhe-rs::shortint. Except for `unchecked_add`, we perform all the operations in the `default` mode. This mode ensures predictable timings along the entire circuit by clearing the carry space after each operation. The configuration is Concrete FFT + AVX-512.
```
Comment (IceTDrinker) [inline tfhe/docs/getting_started/benchmarks/cpu_benchmarks.md:33]: tfhe-rs does not follow the convention for the name display

**Review by IceTDrinker — COMMENTED** [review]
I'm not clear on what was wanted in terms of data display but it looks like we are currently embedding tables which are not related to a specific version of TFHE-rs, I don't think I'm a fan because we need to maintain several sources of data

File: tfhe/docs/getting_started/benchmarks/cpu_benchmarks.md (hunk 6 above)
Comment (yuxizama) [reply tfhe/docs/getting_started/benchmarks/cpu_benchmarks.md:33]: Do you mean tfhe-rs::shortint ?

File: tfhe/docs/references/fine-grained-apis/shortint/parameters.md (hunk 2 above)
Comment (yuxizama) [reply tfhe/docs/references/fine-grained-apis/shortint/parameters.md:37]: Could you suggest the wording change, please?

**Pull request merged**